### DISCUSSION TITLE
Ability to prevent users from editing their profile

### DISCUSSION THREAD

**Discussion by TheMoodBoardz** [snipe/snipe-it#14226] [Ideas]
We use our Snipe-IT build to manage access and as a staff directory however its been noticed that some staff members are going into edit their own profile. It would be great if we could prevent them from editing their own profile as currently its policy to submit changes via our IT/HR team to ensure its applied everywhere it needs to.

**Comment by designatedsuccessor** [comment]
We have the same issue.  We are trying to at least hide it via CSS, but the CSS in Snipe-IT is too ambiguous to properly do this.  The people who coded Snipe-IT inexplicably aren't using IDs on unique elements, which makes targeting more difficult.

**Reply by snipe** [reply]
IDs are not typically required on every element, and this would be an insecure approach to disallowing a user to edit their profile anyway.

If one of the two of you can open a GH issue (not discussion) requesting this feature, we will look into it. Be specific about what you want to disable. If a user forgets their password, should they still have the right to reset it? Can they change their own language? Etc.

**Comment by designatedsuccessor** [comment]
1. I agree that it's not the best approach, but we are trying to make the best out of the situation.
2. Wasn't this already raised here and already closed/resolved? https://github.com/snipe/snipe-it/issues/1845
I don't mind opening again, but I've traditionally avoided opening already-opened issues.

**Reply by snipe** [reply]
I’d open a new ticket (as requested), since the initial purpose of that original issue was met many years ago (and other parts were superseded when we added LDAP integration, which prevents users from changing their password.)

The question I posited there in the original issue still sorta remains, and we have never gotten consensus on is - how locked down do we want to make that? Picture? Location? Language? Email? What things need to be constrained - for each organization, it seems a little bit different.

**Reply by designatedsuccessor** [reply]
I've added a new issue.  To briefly answer some of that here:
- We don't use profile pictures at all.  We do use them in our ERP and Microsoft 365, but that's it.
- At least in our case, we would be fine removing access to /account/profile utterly and completely.  We want one source of authority, and that source would be Microsoft 365 for us -- via SCIM.
